browser/widevine/widevine_utils.cc

```diff
@@ -120,3 +121,13 @@ void InstallBundleOrRestartBrowser() {
   }
 }
 #endif
+
+void DontAskWidevineInstall(content::WebContents* web_contents, bool dont_ask) {
+  Profile* profile = static_cast<Profile*>(web_contents->GetBrowserContext());
+  profile->GetPrefs()->SetBoolean(kDontAskWidevineInstall, dont_ask);
+}
+
+bool ShouldAskWidevineInstall(content::WebContents* web_contents) {
```

Comment:
I'd prefer not to add these methods... This one is used only once, and it is even shorter to directly query prefs in that place

Reply:
Yup, deleted.